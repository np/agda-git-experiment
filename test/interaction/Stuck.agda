module Stuck where

postulate
  I   : Set
  i j : I

data D : I → I → Set where
  d : D i i
  e : D j j

f : ∀ {x} → D i x → Set₁
f p = {!!}
