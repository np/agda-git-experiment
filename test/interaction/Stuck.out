Agda2> (agda2-status-action "")
(agda2-info-action "*Type-checking*" "" nil)
(agda2-highlight-clear)
(agda2-info-action "*Type-checking*" "Checking Stuck (Stuck.agda).\n" t)
(agda2-info-action "*Type-checking*" "Finished Stuck.\n" t)
(agda2-status-action "")
(agda2-info-action "*All Goals*" "?0 : Set₁\n" nil)
((last . 1) . (agda2-goals-action '(0)))
Agda2> (agda2-info-action "*Error*" "Stuck.agda:12,7-11\nI'm not sure if there should be a case for the constructor e,\nbecause I get stuck when trying to solve the following unification\nproblems (inferred index ≟ expected index):\n  j ≟ i\n  j ≟ x\nwhen checking that the expression ? has type Set₁" nil)
((last . 3) . (agda2-goto '("Stuck.agda" . 137)))
(agda2-status-action "")
Agda2> 